[2025-07-22T10:06:44.304+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-22T10:06:44.335+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_postgres.load_time_dim_table manual__2025-07-22T09:44:34.337847+00:00 [queued]>
[2025-07-22T10:06:44.351+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_postgres.load_time_dim_table manual__2025-07-22T09:44:34.337847+00:00 [queued]>
[2025-07-22T10:06:44.352+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-07-22T10:06:44.372+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_time_dim_table> on 2025-07-22 09:44:34.337847+00:00
[2025-07-22T10:06:44.383+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1031) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-22T10:06:44.386+0000] {standard_task_runner.py:72} INFO - Started process 1033 to run task
[2025-07-22T10:06:44.383+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_postgres', 'load_time_dim_table', 'manual__2025-07-22T09:44:34.337847+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmppijhxjn7']
[2025-07-22T10:06:44.388+0000] {standard_task_runner.py:105} INFO - Job 34: Subtask load_time_dim_table
[2025-07-22T10:06:44.498+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_postgres.load_time_dim_table manual__2025-07-22T09:44:34.337847+00:00 [running]> on host 0c6a6447a272
[2025-07-22T10:06:44.664+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_postgres' AIRFLOW_CTX_TASK_ID='load_time_dim_table' AIRFLOW_CTX_EXECUTION_DATE='2025-07-22T09:44:34.337847+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-22T09:44:34.337847+00:00'
[2025-07-22T10:06:44.668+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-22T10:06:45.060+0000] {etl_dag.py:105} INFO - Loading time dimension table
[2025-07-22T10:06:45.065+0000] {etl_dag.py:108} INFO - Time dimension table loaded successfully
[2025-07-22T10:06:45.067+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-22T10:06:45.090+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-22T10:06:45.093+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_postgres, task_id=load_time_dim_table, run_id=manual__2025-07-22T09:44:34.337847+00:00, execution_date=20250722T094434, start_date=20250722T100644, end_date=20250722T100645
[2025-07-22T10:06:45.166+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-22T10:06:45.182+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
